portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,261 @@
 <!DOCTYPE html>
-<html>
-  <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
-  </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
-  </body>
+<html lang ="en">
+<head>
+<title>Priya Kaushal</title>
+<link href="style.css" rel="stylesheet" type="text/css">
+<!-- mobile friendly -->
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<meta charset="utf-8">
+</head>
```

Comment:
It would be nice to have indentation for tags at different levels. Helps with reading the code. I would make it like:

```
<head>
    <title>...</title>
    <link.......>
    <meta .....>
   ....
</head>
```